src/identity-wallet.js

```diff
@@ -41,7 +41,6 @@ class IdentityWallet {
   }
 
   hasConsent (spaces = [], origin) {
-    if (!origin) return true
     const prefix = `3id_consent_${this._keyring.getPublicKeys().managementKey}_${origin}_`
```

Comment:
if not passed, still in prefix?

Reply:
good point. But should always be `null` if not passed.